jsonify_stored/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2022 Camptocamp SA (http://www.camptocamp.com).
+# @author Simone Orsi <simone.orsi@camptocamp.com>
+# @author Matthieu Méquignon <matthieu.mequignon@camptocamp.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+{
+    "name": "JSONify Stored",
```

Comment:
As it depends on `base_jsonify`, maybe this module could be named `base_jsonify_stored`

Reply:
actually I'd like to change the name of base_jsonify to `jsonify` or `jsonifier` because: 
1. there's a conflict on odoo apps store
2. there's nothing "basic" here: it's a tool that you can use as it is on every model w/ out having to code anything